blueprint/core/blueprint_GenericEditor.cpp

```diff
@@ -141,20 +149,14 @@ namespace blueprint
 
     void BlueprintGenericEditor::registerAppRootCallbacks()
     {
-        appRoot.beforeBundleEval = [=] (const juce::File& bundle)
+        harness.beforeBundleEval = [=] (std::shared_ptr<blueprint::EcmascriptEngine> engine, const juce::File& bundle)
```

Comment:
Again. Maybe just kill `juce::File` for the default `HotReloadHarness` callbacks?